packages/connectors/connector-dingtalk-web/src/index.ts

```diff
@@ -80,7 +79,6 @@ export const getAccessToken = async (
   }
 
   const { accessToken } = result.data;
-  assert(accessToken, new ConnectorError(ConnectorErrorCodes.InvalidResponse));
```

Comment:
Can take a look on the definition of the `accessTokenResponseGuard`, `accessToken` is now required. No need to further check the existence of `accessToken`.